pkgs/development/python-modules/simple-term-menu/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pyproject = true;
+  nativeBuildInputs = [ setuptools ];
+
+  disabled = pythonOlder "3.4";
+
+  pname = "simple-term-menu";
+  version = "1.6.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "be9c5dbd8df12a404b14cd8e95d6fc02d58c60e2555f65ddde41777c487fb3b9";
```

```suggestion
    hash = "sha256-vpxdvY3xKkBLFM2Oldb8AtWMYOJVX2Xd3kF3fEh/s7k=";
```
Could you please use SRI hash instead?
https://nixos.org/manual/nixpkgs/stable/#fetchurl